slurm_eval.sh

```diff
@@ -16,18 +16,21 @@ export NORMALIZE=false
 conda activate neural-lam
 
 if [ "$PREPROCESS" = true ]; then
+    echo "Create static features"
     srun -ul -N1 -n1 python create_static_features.py --boundaries 60
+    echo "Creating mesh"
     srun -ul -N1 -n1 python create_mesh.py --dataset "cosmo" --plot 1
+    echo "Creating grid features"
     srun -ul -N1 -n1 python create_grid_features.py --dataset "cosmo"
     if [ "$NORMALIZE" = true ]; then
         # This takes multiple hours!
+        echo "Creating normalization weights"
         srun -ul -N1 -n1 python create_parameter_weights.py --dataset "cosmo" --batch_size 32 --n_workers 8 --step_length 1
     fi
 fi
 
 ulimit -c 0
 export OMP_NUM_THREADS=16
 
-# Run the script with torchrun
 srun -ul python train_model.py --load "wandb/example.ckpt" --dataset "cosmo" \
-    --eval="test" --subset_ds 1 --n_workers 2 --batch_size 6 --wandb_mode "offline"
```

Comment:
why can we switch off offline now?

Reply:
As shown in the `README.md` it is preferred to give the option to the user whether to `wandb login` or `wandb off`. I am not 100% sure how this setting interacts with slurm/sbatch scripts though.